pkg/dns/ibm/common.go

```diff
@@ -50,3 +78,21 @@ func ValidateInputDNSData(record *iov1.DNSRecord, zone configv1.DNSZone) error {
 	}
 	return kerrors.NewAggregate(errs)
 }
+
+// GetCustomServiceEndpointURL return the url of serviceType from endpoints if exist or else returns the default url
+func GetCustomServiceEndpointURL(endpoints []ServiceEndpoint, serviceType string) (string, error) {
```

Comment:
I can update `GetCustomServiceEndpointURL` to `GetServiceEndpointURL`, But would like to keep it Name as it as instead of `CISCustomEndpointName` so it will be in align with the [apis](https://github.com/Karthik-K-N/cluster-ingress-operator/blob/be08afc7f875d98d48faf17e88c015451082c00b/vendor/github.com/openshift/api/config/v1/types_infrastructure.go#L709-L731).